nemo_text_processing/inverse_text_normalization/en/taggers/date.py

```diff
@@ -64,6 +64,15 @@ def _get_range_graph():
     return graph
 
 
+def _get_financial_period_graph():
+    h_ordinals = pynini.cross('first', '1') | pynini.cross('second', '2')
+    q_ordinals = h_ordinals | pynini.cross('third', '3') | pynini.cross('fourth', '4')
+
+    h_graph = h_ordinals + pynini.cross(' half', 'H') | pynini.cross(' h', 'H')
+    q_graph = q_ordinals + pynini.cross(' quarter', 'Q') | pynini.cross(' q', 'Q')
```

Comment:
could you move these words to files, please?

Reply:
Moved